src/battle_script_commands.c

```diff
@@ -5349,14 +5349,14 @@ static bool32 TryKnockOffBattleScript(u32 battlerDef)
     return FALSE;
 }
 
-#define SYMBIOSIS_CHECK(battler, ally)                                                                                               \
```

Comment:
I agree with Alex here, it's more likely the checks pass most of the time, so it's better to check the ability first.